### PR TITLE
#5: ODP : Return Types: Service/repository

### PR DESCRIPTION
**Pull request by sorja**
Part of #806

### CODE REVIEW

File: server/repository/odp/markAsActual.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { insertAudit } from '@server/repository/audit/auditRepository'
 import { OdpClassRepository } from '@server/repository'
 import { getAndCheckOdpCountryId } from './getAndCheckOdpCountryId'
 
-export const markAsActual = async (client: any, odpId: any, user: any) => {
+export const markAsActual = async (client: any, odpId: any, user: any): Promise<null | [void, void]> => {
```
Comment (sorja) [inline server/repository/odp/markAsActual.ts:5]: Not sure about this? Thoughts?

File: server/repository/odp/getOdp.ts [hunk 2]
```diff
@@ -5,9 +5,10 @@ import * as camelize from 'camelize'
 import * as db from '@server/db/db_deprecated'
 
 import { OdpClassRepository } from '@server/repository'
+import { ODP } from '@core/odp'
 import { getOdpVersionId } from './getOdpVersionId'
 
-export const getOdp = async (odpId: any, schemaName = 'public') => {
+export const getOdp = async (odpId: any, schemaName = 'public'): Promise<ODP> => {
```
Comment (minotogna) [inline server/repository/odp/getOdp.ts:11]: no any

File: server/repository/odp/markAsActual.ts (hunk 1 above)
Comment (minotogna) [reply server/repository/odp/markAsActual.ts:5]: what about Promise<void> only?
and instead of returning null, you can return Promise.resolve() ?

Comment (sorja) [reply server/repository/odp/markAsActual.ts:5]: Can't return only promise: `TS2314: Generic type 'Promise ' requires 1 type argument(s).`

Comment (sorja) [reply server/repository/odp/markAsActual.ts]: Resolved in next pr

**Review by minotogna — APPROVED** [review]

**Pull request merged**